scripts/flash.sh

```diff
@@ -25,3 +35,14 @@ fi
 
 cargo build --release --manifest-path libs/intel-spi/Cargo.toml
 sudo libs/intel-spi/target/release/intel-spi "build/${MODEL}/coreboot.rom"
+
+if [ "$WITH_EC" = true ]
+then
+    if [ -e "${MODEL_DIR}/ec.config" ]
+    then
+        source "${MODEL_DIR}/ec.config"
+        make -C ec BOARD=${BOARD} VERSION=${VERSION} flash_internal
```

Comment:
Instead of doing it like this, I would recommend doing the following:

```
if [ -e "build/${MODEL}/ec.rom" ]
then
    pushd ec
    ./ectool.sh flash_internal "../build/${MODEL}/ec.rom"
    popd
done
```

Reply:
That way it will use the ec.rom that was built for that model, and you don't have to calculate VERSION

Reply:
An alternative to using ectool.sh and pushd is to do it like the intel-spi stuff above:

```
cargo build --release --manifest-path ec/tool/Cargo.toml
sudo ec/tool/target/release/system76_ectool "build/${MODEL}/ec.rom"
```